### PR TITLE
[T5A2][F10-B2]Ravi Shwetha

### PR DESCRIPTION
**Pull request by ravishwetha**
Ready for Review

### CODE REVIEW

File: src/seedu/addressbook/commands/AddCommand.java [hunk 1]
```diff
@@ -12,6 +12,8 @@
  * Adds a person to the address book.
  */
 public class AddCommand extends Command {
+	
+	public static final boolean isMutating = true;
```
Comment (KulaniT) [inline src/seedu/addressbook/commands/AddCommand.java]: alignment issue

File: src/seedu/addressbook/commands/ClearCommand.java [hunk 2]
```diff
@@ -4,6 +4,8 @@
  * Clears the address book.
  */
 public class ClearCommand extends Command {
+	
+	public static final boolean isMutating = true;
```
Comment (KulaniT) [inline src/seedu/addressbook/commands/ClearCommand.java]: why public?

File: src/seedu/addressbook/commands/AddCommand.java [hunk 3]
```diff
@@ -67,4 +69,9 @@ public CommandResult execute() {
         }
     }
 
+	@Override
+	public boolean isMutating() {
+		return isMutating;
```
Comment (KulaniT) [inline src/seedu/addressbook/commands/AddCommand.java]: Your approach is good . But you can simply return true or false from the method in this case.

**Comment by KulaniT** [comment]
@ravishwetha : Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/commands/ClearCommand.java (hunk 2 above)
Comment (ravishwetha) [inline src/seedu/addressbook/commands/ClearCommand.java]: Yup, it should be private since there is technically a getter function

**Pull request closed without merging**